mlrun/config.py

```diff
@@ -284,6 +285,29 @@ def dbpath(self, value):
             # when dbpath is set we want to connect to it which will sync configuration from it to the client
             mlrun.db.get_run_db(value)
 
+    @property
+    def iguazio_api_url(self):
+        if not self._iguazio_api_url:
+            # we want to be able to run with old versions of the service who runs the API (which doesn't configure this
+            # value) so we're doing best effort to try and resolve it from other configurations
```

Comment:
- to docstring (long)
- this is a hack - add TODO to remove once enough time / versions pass